agendas/2019-11-07.md

```diff
@@ -56,4 +57,5 @@ Howard Lewis Ship    | Walmart             | Portland, OR
 1. Input Union [RFC](https://github.com/graphql/graphql-spec/blob/master/rfcs/InputUnion.md) (30m)
 1. Custom Scalar Specification [RFC](https://github.com/graphql/graphql-spec/issues/635) (10m)
 1. Review implementation complexity of [Disallow non-breakable chains of circular references in Input Objects](https://github.com/graphql/graphql-spec/pull/445) (10m)
+1. Facebook CLA in repos (5m)
```

Comment:
@ErikWittern Thanks for bringing it up 👍 
Couple WG ago we agreed that every agenda item should have linked to some sort of issue/PR/RFC document so we can track it and have post-WG discussions.
It's totally non-critical so I merged this one but can you please open an issue on this repo and link the agenda item to it?

Reply:
Ah, sorry I did not do so before. I created an issue (https://github.com/graphql/graphql-wg/issues/291) and filed a new PR to address this (https://github.com/graphql/graphql-wg/pull/292).